requirements.txt

```diff
@@ -1,7 +1,7 @@
 ansible>=2.9.0
 os-client-config
 shade
-openstacksdk
+openstacksdk<=0.45.0; python_version<='2.7'
```

Comment:
openstacksdk [release notes](https://docs.openstack.org/releasenotes/openstacksdk/ussuri.html#relnotes-0-46-0-stable-ussuri) state:
> openstacksdk does not test or support python2 as of 0.40, but the releases have still accidentally worked (except for 0.44 which was broken for python2).

So should this be either `==0.40.0` (strictly) or at least `==0.45.0` given we must be fairly confident that works?

Reply:
I will change this to ==0.45.0, 0.45.0 is the last supported version according to pip-requires.